app/src/main/java/org/p2p/wallet/home/HomeModule.kt

```diff
@@ -43,7 +44,7 @@ object HomeModule : InjectionModule {
     private fun Module.initDataLayer() {
         factory<HomeLocalRepository> { HomeDatabaseRepository(get()) }
         factoryOf(::HomeMapper)
-        factory<RefreshErrorRepository> { RefreshErrorSharedFlowRepository() }
+        factoryOf(::RefreshErrorSharedFlowRepository) bind RefreshErrorRepository::class
```

Comment:
We name impl of a repository using Remote/Local/InMemory words. 
If it caches values - you can name it RefreshErrorInMemoryRepository: RefreshErrorLocalRepository